### PR TITLE
Lists

### PR DESCRIPTION
**Pull request by jdungan**
Starting to add the basic functions of seeing your Bids, Votes, etc.  I copied the claim_status pattern but this seems like a lot of work.  I'm open to suggestion on a cleaner way to show lists of objects.

### CODE REVIEW

File: api/templates/bid.html [hunk 1]
```diff
@@ -2,7 +2,12 @@
 
 {% block widget_content %}
 
-{% include "includes/claim_form.html" %}
+{% if bid.ask_met %}
```
Comment (groovecoder) [inline api/templates/bid.html]: Could be `{% if bid.ask_met or claim %}` ?

File: api/views.py [hunk 2]
```diff
@@ -114,3 +114,47 @@ class VoteViewSet(AutoOwnObjectsModelViewSet):
...
+    """
+    renderer_classes = (TemplateHTMLRenderer,)
+
+    def get(self, request, format=None):
+        try:
+            votes = Vote.objects.filter(user=self.request.user)
+        except:
+            votes = []
+
+        return Response({'votes': votes}, template_name='vote_list.html')
```
Comment (groovecoder) [inline api/views.py]: Could/should still keep these separate `APIView` classes, but we may be able to make a mixin to reduce some code duplication here.

File: auctions/models.py [hunk 3]
```diff
@@ -92,8 +99,21 @@ def __unicode__(self):
...
 
+@receiver(post_save, sender=Issue)
+def lookup_title(sender, instance, **kwargs):
+    try:
+        r = requests.get(instance.url)
+        title_search = re.search('(?:<title.*>)(.*)(?:<\/title>)', r.text)
+        if title_search:
+            title = HTMLParser.HTMLParser().unescape(title_search.group(1))
+            Issue.objects.filter(id=instance.id).update(title=title)
+    except:
```
Comment (groovecoder) [inline auctions/models.py]: blank `except:` should catch a specific exception.

File: auctions/models.py [hunk 4]
```diff
@@ -92,8 +99,21 @@ def __unicode__(self):
...
+        pass
+
+
 class Claim(models.Model):
     STATUS_CHOICES = (
+        ('Submitted', 'Submitted'),
```
Comment (groovecoder) [inline auctions/models.py]: Not sure we actually need a `Submitted` status apart from `Pending` ?

File: auctions/tests/model_tests.py [hunk 5]
```diff
@@ -42,12 +42,24 @@ def _add_url(self, string):
         return string.format(url=self.url)
 
 
-class BidTests(MarketTestCase):
+class IssueTest(MarketTestCase):
+
+    def test_unicode(self):
```
Comment (groovecoder) [inline auctions/tests/model_tests.py]: :100:

File: codesy/templates/base.html [hunk 6]
```diff
@@ -16,13 +16,14 @@
...
+  <div class="nav-bar top-bar">
           <a href="/"><img src="{% static "img/codesy_300dpi_323x80.png" %}" width="161" height="40"/></a>
 {% if user.is_authenticated %}
          <div class="top-bar-right">
             <ul class="menu">
-               <li><a href="">My Claims...</a></li>
-<!-- TODO: Create menus for claims, bids, votes, etc. -->
+               <li><a href="/bid-list">Bids</a></li>
+               <li><a href="/claim-list">Claims</a></li>
+               <li><a href="/vote-list">Votes</a></li>
```
Comment (groovecoder) [inline codesy/templates/base.html]: We should turn this into a single drop-down menu.

**Comment by groovecoder** [comment]
List views look good.

File: auctions/models.py (hunk 4 above)
Comment (jdungan) [inline auctions/models.py]: Submitted is for a claim without votes.  Pending means some votes but not all.  Not sure if this is an important distinction.

File: api/templates/bid.html (hunk 1 above)
Comment (jdungan) [inline api/templates/bid.html]: c12ac0d

File: api/views.py (hunk 2 above)
Comment (jdungan) [inline api/views.py]: #287

**Pull request merged**